src/main/resources/config/application-dev.yml

```diff
@@ -25,10 +25,16 @@ vas:
     jwk-set-uri: https://centralidp.dev.demo.catena-x.net/auth/realms/CX-Central/protocol/openid-connect/certs
     auth-url: https://centralidp.dev.demo.catena-x.net/auth/realms/CX-Central/protocol/openid-connect/auth
 
+spring:
+  config:
+    activate:
+      on-profile: dev
 
 
 application:
   name: dev
   host: http://project-value-added-service-vas/api/schedule
   partnersPoolUrl:
-    legalEntities: 'https://partners-pool.dev.demo.catena-x.net/api/catena/legal-entities'
\ No newline at end of file
+    bpdmAddressUrl: 'https://business-partners.dev.demo.catena-x.net/companies/test-company/api/catena/output/addresses/search?limit=10'
```

Comment:
Do you really need to specify this in the `application.yaml`? These are hard coded URLs to systems, that are not available for everyone. 
So please, if you are deploying your application, provide this as environment variables (spring boot picks that up for you) or any other option